### PR TITLE
Changelog for 0.9.0

### PR DESCRIPTION
**Pull request by JessicaJHee**

### CODE REVIEW

**Comment by angelozerr** [comment]
Please update pom.xml version too

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # MicroProfile Language Server Changelog
 
+## [0.9.0](https://github.com/eclipse/lsp4mp/milestone/10?closed=1) (August 3, 2023)
```
Comment (datho7561) [inline CHANGELOG.md:3]: Update date to August 8.

**Review by datho7561 — APPROVED** [review]
Looks good!

File: CHANGELOG.md (hunk 1 above)
Comment (JessicaJHee) [reply CHANGELOG.md:3]: will update for all the changelogs :+1:

**Pull request merged**